model/riscv_softfloat_interface.sail

```diff
@@ -38,9 +37,11 @@ type bits_L      = bits(64)   /* Signed integer */
 type bits_LU     = bits(64)   /* Unsigned integer */
 
 /* ***************************************************************** */
-/* Internal registers to pass results across the softfloat interface
- * to avoid return types involving structures.
- */
+/* Internal registers to pass results across the softfloat interface */
+/* to avoid return types involving structures.                       */
+/*                                                                   */
+/* Todo: Return the float_result directly in Sail.                   */
+/* ***************************************************************** */
```

Comment:
I don't think you should be touching this? Your new code doesn't have this issue.

Reply:
Just hint here to remind that, the float in saill will return directly, as well as the softfloat_interface will be retired soon.

I can revert this change if you have any concern here.

Reply:
Reverted this change.